packages/bruno-electron/src/utils/common.js

```diff
@@ -137,6 +137,10 @@ const parseDataFromRequest = (request) => {
   return parseDataFromResponse(requestCopy);
 };
 
+const isValidValue = (val) => {
```

Comment:
I'm concerned that placing this function in `common/utils` might lead others to reuse it beyond encryption-related purposes, and potentially updating the logic without recognizing the encryption-specific nuances involved.

I see two approaches that I feel are better
1. Explicitly export `isValueEncryptable` and `isValueDecryptable` from `utils/encryption.js`. It's a bit redundant but ensures clarity and protects against future misuse.
2. Adjust `encryptString` and `decryptString` to simply return an empty string when one is passed.

Ex:
```javascript
function encryptString(str) {
  if (typeof str === 'string' && str.length === 0) {
    return ''
  }
```